tools/cli/internal/openapi/filter/extension_test.go

```diff
@@ -77,6 +77,80 @@ func TestXSunsetFilter_removeSunset(t *testing.T) {
 	}
 }
 
+func TestExtensionFilter_removeIpaException(t *testing.T) {
```

Comment:
some of these seem like test cases see for how we do test cases in go https://github.com/mongodb/openapi/blob/e1a969ddad16cd8f4df6dc45c5625c9bd464102a/tools/cli/internal/apiversion/version_test.go#L25-L30